integration-tests/tests/fdw.rs

```diff
@@ -4,6 +4,7 @@
 // http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
+#![cfg(never)]
```

Comment:
Is it possible to make this conditional on `pg-extend`'s features?

Reply:
Yes, I thought there was a flag for this, for now, how about `#![cfg(not(feature = "postgres-12"))]`